src/gui/creds/httpcredentialsgui.cpp

```diff
@@ -13,10 +13,12 @@
  * for more details.
  */
 
-#include "application.h"
+#include "creds/httpcredentialsgui.h"
 #include "account.h"
+#include "application.h"
 #include "common/asserts.h"
-#include "creds/httpcredentialsgui.h"
+#include "httploginrequiredwidget.h"
```

Comment:
Please use the full path gui/loginreq...

I'd like to add a bit more structure to the includes, at least for new code.